extendeddatacrossword_test.go

```diff
@@ -390,8 +390,13 @@ func TestCorruptedEdsReturnsErrByzantineData_UnorderedShares(t *testing.T) {
 
 	codec := NewLeoRSCodec()
 
+	edsWidth := 4            // number of shares per row/column in the extended data square
+	odsWidth := edsWidth / 2 // number of shares per row/column in the original data square
+	registerTree("Testing", newConstructor(uint64(odsWidth), nmt.NamespaceIDSize(namespaceSize)))
```

Comment:
[nit] to conform to some naming convention around custom tree names

```suggestion
	registerTree("test-tree", newConstructor(uint64(odsWidth), nmt.NamespaceIDSize(namespaceSize)))
```